src/main/java/uk/gov/hmcts/darts/event/service/impl/RemoveDuplicateEventsProcessorImpl.java

```diff
@@ -58,54 +33,25 @@ public RemoveDuplicateEventsProcessorImpl(
 
     @Override
     @Transactional
-    public void processRemoveDuplicateEvents() {
-        if (clearUpWindow < 1) {
-            throw new DartsException(
-                "The duplicate event deletion task is not configured correctly. " +
-                    "The clear-up window must be at least 1 day but is configured as " + clearUpWindow);
+    public boolean processEvent(Integer eventId) {
```

Comment:
Maybe this is abit subjective but is the method named appropriately here? Agreed, we are processing the event, but do you think it may be confusing to return `true` if a duplicate is found? Like the logic in `AsyncEventProcessor` only continues if this processEvent returns `false`, which seems wrong. 
Wonder if it should be renamed to something like `findAndRemoveDuplicateEvent` ? Or something along those lines.